web/src/pages/project/ProjectGrid.tsx

```diff
@@ -1,7 +1,13 @@
 import * as React from 'react'
 import _ from 'lodash'
-import { Table } from 'semantic-ui-react'
+import { Table, Form, Popup } from 'semantic-ui-react'
```

Comment:
This might've changed from my PR, using a different Table. 
Edit: On reflection you need to merge from main)